src/ipc/ipc3/handler.c

```diff
@@ -1505,6 +1505,15 @@ void ipc_boot_complete_msg(ipc_cmd_hdr *header, uint32_t *data)
 	*header = SOF_IPC_FW_READY;
 }
 
+void ipc_msg_reply(struct sof_ipc_reply *reply)
+{
+	struct ipc *ipc = ipc_get();
+
+	ipc->delayed_response = false;
```

Comment:
This may need locking with IRQs OFF - see #4715

Reply:
sure, will rebase it after 4715 is merged